src/main/scala/com/github/awant/habrareader/models/Post.scala

```diff
@@ -15,38 +17,40 @@ case class Post(link: String,
                 viewsCount: Int,
                 commentsCount: Int,
                 bookmarksCount: Int,
-                updateDate: Date)
+                updateDate: Date) {
+  val id: Long = link.filter(_.isDigit).toLong
```

Comment:
Why do you need this?
It is unsafe to do in any case.
If you need id, add in Post constructor